provisioner/influx-series.js

```diff
@@ -39,6 +39,78 @@ module.exports.spotRequestsSubmitted = new base.stats.Series({
   },
 });
 
+// Store the times when a spot request is fulfilled
+module.exports.spotRequestsFulfilled = new base.stats.Series({
+  name: 'AwsProvisioner.SpotRequestsFulfilled',
+  columns: {
+    provisionerId: base.stats.types.String,
+    region: base.stats.types.String,
+    az: base.stats.types.String,
+    instanceType: base.stats.types.String,
+    workerType: base.stats.types.String,
+    id: base.stats.types.String,
+    instanceId: base.stats.types.String,
```

Comment:
This discussion is relevant to everything... but assuming we don't make 1k spot reqs a minutes, the load impossed by this data is probably not important.. So leaving the strings is probably a sound approach to this :)